src/extension.ts

```diff
@@ -117,17 +140,32 @@ function initRPC(clientID: string): void {
 	rpc.login(clientID).catch(async error => {
 		if (reconnectTimer) {
 			// Destroy and dispose of everything after a default of 20 reconnect attempts
-			if (reconnectCounter >= config.get('reconnectThreshold')) await destroyRPC();
-			else return;
+			if (reconnectCounter >= config.get('reconnectThreshold')) {
+				createButon();
+				await destroyRPC();
+			} else return;
```

Comment:
If you use curlies for the if also use it for the else like you do in the createButton function below.